### PR TITLE
Catalog: Rename the json file to reflect the id and connectors.properties content

### LINKED ISSUE #534: Catalog: Rename the json file to reflect the id and connectors.properties content
In connectors.properties we have the following:

camel-aws2-s3-source
camel-aws2-s3-sink

while in the connectors folder

camel-aws2-s3-kafka-sink-connector.json            
camel-aws2-s3-kafka-source-connector.json  

And inside these files the id is:

camel-aws2-s3-sink
camel-aws2-s3-source

so it probably makes sense to rename the json as something like <id>.json, to make the catalog easier to use.

Related to #430

### PR DESCRIPTION
**Pull request by oscerd**
Fixes #534

### CODE REVIEW

File: camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		"camel.sink.endpoint.cookieHandler": {
+			"name": "camel.sink.endpoint.cookieHandler",
+			"description": "Configure a cookie handler to maintain a HTTP session",
+			"defaultValue": "null",
+			"priority": "MEDIUM"
+		},
+		"camel.sink.endpoint.headerFilterStrategy": {
+			"name": "camel.sink.endpoint.headerFilterStrategy",
+			"description": "To use a custom HeaderFilterStrategy to filter header to and from Camel message.",
+			"defaultValue": "null",
```
Comment (apupier) [inline camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json:44]: Isn't a problem to make the difference between the `null` as `String` and no default value?

I'm guessing here that it means there is no default value defined

Comment (oscerd) [reply camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json:44]: Yes, there is no default value when it's null. This is something coming from the how kafka connect define the options.

Comment (apupier) [reply camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json:44]: it means that we cannot define a default value which is the null String? or there are ways to escape it?

Comment (oscerd) [reply camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json:44]: I think we can improve the connector aurogeneration for this. @valdar cc

Comment (valdar) [reply camel-kafka-connector-catalog/src/generated/resources/connectors/camel-ahc-sink.json:44]: @oscerd it seems that we can use `ConfigDef.NO_DEFAULT_VALUE` kafka connect side to distinguish between no default value and `null` as default value. I am investigating if that information is available in the vanilla camel catalog that we process to generate connectors.

**Pull request merged**